packages/big-design/src/components/MultiSelect/spec.tsx

```diff
@@ -106,21 +106,23 @@ const MultiSelectWithOptionsDescriptions = (
 );
 
 test('renders select combobox', async () => {
-  const { getByRole } = render(MultiSelectMock);
```

Comment:
Changed all (I hope) of the tests to use `screen`. However, there might have been an exception or two.